coded_tools/agent_network_designer/get_agent_network.py

```diff
@@ -63,3 +62,9 @@ def invoke(self, args: Dict[str, Any], sly_data: Dict[str, Any]) -> Union[Dict[s
         logger.info("The resulting agent network: \n %s", str(the_agent_network_str))
         logger.info(">>>>>>>>>>>>>>>>>>>DONE !!!>>>>>>>>>>>>>>>>>>")
         return the_agent_network_str
+
+    async def async_invoke(self, args: Dict[str, Any], sly_data: Dict[str, Any]) -> str:
```

Comment:
`CodedTool` is an abstract class, so we must implement this method. For now I just delegate to the sync method.
Did that for the other CodedTools too.